parser/parser/taxonomy_parser.py

```diff
@@ -0,0 +1,207 @@
+from neo4j import GraphDatabase
```

Comment:
you don't need to call this file taxonomy_parser.py, parser.py is enough, the problem you had is because you don't have a __init__.py file in you parser directory.

If you add it, in test, you will be able to write:
`from parser import parser`

that said I think we should name the package taxonomy_parser (the directory) and the file should be parser.py

This is because you have to think you package name as the published name whchi should be quite unique.
(even openfoodfacts_taxonomy_parser ?)